hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2892,9 +2892,9 @@ func reconcileAutoScalerDeployment(deployment *appsv1.Deployment, hc *hyperv1.Ho
 		// we might end up locked with three nodes.
 		"--skip-nodes-with-local-storage=false",
 		"--alsologtostderr",
-		"--leader-elect-lease-duration=60s",
-		"--leader-elect-retry-period=15s",
-		"--leader-elect-renew-deadline=40s",
+		"--RecommendedLeaseDuration=137s",
```

Comment:
typo? need to do `fmt.Sprintf("--leader-elect-lease-duration=%s", config.RecommendedLeaseDuration)`, basically same than in https://github.com/openshift/hypershift/pull/1465/files#diff-b5cde720842b19b53da3f454d827e27327c4559b2e4849906dac728586f51eb3R2646